webrtc.html

```diff
@@ -1837,61 +1808,32 @@ <h4>Set the RTCSessionDescription</h4>
-                                <var>transceiver</var> and <var>muteTracks</var> and set
-                                <a>[[\FiredDirection]]</a> to <code>"inactive"</code>.</p>
-                              </li>
-                              <li>
-                                <p><a data-lt="stop the RTCRtpTransceiver">Stop the
-                                RTCRtpTransceiver</a> <var>transceiver</var>.</p>
-                              </li>
-                              <li>
-                                <p>Remove <var>transceiver</var> from <var>connection</var>'s
-                                <a href="#transceivers-set">set of transceivers</a>.</p>
-                              </li>
-                            </ol>
-                          </li>
                           <li>
-                            <p>Otherwise, (if the <var>transceiver</var> was not just removed)
-                            run the following steps:</p>
+                            <p>If <var>remote</var> is <code>true</code>, run the following sub steps:</p>
                             <ol>
                               <li>
-                                <p>If the <var>transceiver</var>'s <a>[[\FiredDirection]]</a>
-                                is either <code>"sendonly"</code> or <code>"inactive"</code>
-                                and <var>transceiver</var>'s <a>[[\CurrentDirection]]</a> is
-                                either <code>"sendrecv"</code> or <code>"recvonly"</code>,
-                                or the "set the associated remote streams" step above increased
-                                the length of <var>addList</var>, 
-                                <a>process the addition of a remote track</a> with
-                                <var>transceiver</var> and <var>trackEventInits</var> and
-                                set the <var>transceiver</var>'s <a>[[\Receptive]]</a> slot
-                                to <code>true</code>.</p>
-                              </li>
-                              <li>
-                                <p>If the <var>transceiver</var>'s <a>[[\FiredDirection]]</a>
-                                is either <code>"sendrecv"</code> or <code>"recvonly"</code>
-                                and <var>transceiver</var>'s <a>[[\CurrentDirection]]</a> is
-                                either <code>"sendonly"</code>, <code>"inactive"</code> or
-                                <code>null</code>, <a>process the removal of a remote track</a>
-                                with <var>transceiver</var> and <var>muteTracks</var> and
-                                set the <var>transceiver</var>'s <a>[[\Receptive]]</a> slot
-                                to <code>false</code>.</p>
+                                <p>Let <var>msids</var> be the a list of MSIDs that
+                                <var>transceiver</var>'s <a>[[\CurrentRemoteDescription]]</a> indicates
```

Comment:
Please copy the [[LastStableFoo]] pattern above and add a [[LastStableAssociatedRemoteMediaStreams]] to the receiver. Parsing SDP could be burdensome if you're implementing this, and having a separate pattern for this one thing might beg the question what's special about it

Reply:
And we [already have it](https://w3c.github.io/webrtc-pc/#dfn-laststablestateassociatedremotemediastreams)! Will update to use it.

Reply:
@henbos And I'll remove the existing use of [[LastStableAssociatedRemoteMediaStreams]] which is incorrect (passes in MediaStream objects where MSIDs are expected). Fyi.